jsk_robot_common/jsk_robot_startup/config/install_systemd.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+jsk_robot_startup=$(builtin cd "`dirname "${BASH_SOURCE[0]}"`"/.. > /dev/null && pwd)
+if [ $# != 1 ]; then
+    echo "Invalid number of arguments"
+    exit 1
+fi
+
+# Copy scripts
+cd $jsk_robot_startup/systemd
+if [ ! -d /usr/local/bin/jsk_robot_startup ]; then
+  sudo mkdir -p /usr/local/bin/jsk_robot_startup
```

Comment:
Could you change this directory to `/usr/local/src` (maybe `/usr/local/share/`) ?
We should not copy these file under the `bin` directory if we do not deal with these file as binary executable.